website/docs/install/config.md

```diff
@@ -572,11 +572,11 @@ Web 版认证 token 和 iOS 内购回执认证 token 只需选择其一填入即
 
 -   `SEHUATANG_COOKIE`: 登陆色花堂后的 cookie 值。
 
-### 网易云歌单
+### Netease Cloud Music Playlists
```

Comment:
Please put in before
https://github.com/DIYgod/RSSHub/blob/bca6b4bc3f3c31e7c4677fdaffe917f67052fed4/website/docs/install/config.md?plain=1#L398